content/.well-known/security.txt

```diff
@@ -0,0 +1,13 @@
+# These web pages have always up to date information how to report issues:
+Contact: https://www.jenkins.io/security/reporting/
+Contact: https://www.jenkins.io/security/#reporting-vulnerabilities
+
+# We also accept emails sent to this address:
+Contact: mailto:jenkinsci-cert@googlegroups.com
+
+# While team members speaks other languages, this is the language common to all
+Preferred-Languages: en
+
+Expires: 2023-01-01T00:00:00.000Z
```

Comment:
Does this need to be updated?  And how do we flag it to ensure that it is always updated?

Reply:
Yes, needs to be updated. Let me see whether there's something I can do about that…

Reply:
For now

```suggestion
Expires: 2026-01-01T00:00:00.000Z
```

I couldn't find a straightforward way to make it happen with Awestruct, might end up being `updatecli`.